### DISCUSSION TITLE
How to disable Next preload meta tag?

### DISCUSSION THREAD

**Discussion by ilhamwahabi** [vercel/next.js#24699] [Help] [answered]
Hi, when using **Next.js 10.0.8** and using "View page source" we get this result in production which is expected. Where the meta tag is displayed how it should be.

![html1](https://user-images.githubusercontent.com/20817629/116800234-7b905f00-ab29-11eb-8625-5cb787a8e4dc.png)

But when using latest Next.js version we get this result that is not expected, although the meta tag functionality still same but I think this can raise issue in other places

![html2](https://user-images.githubusercontent.com/20817629/116800240-9367e300-ab29-11eb-8ea6-ab39c41df5ee.png)

For example: now in PWABuilder my app manifest is not detected anymore where it still worked when my app still using Next 10.0.8

![PWABuilder2](https://user-images.githubusercontent.com/20817629/116800302-1ab55680-ab2a-11eb-8acd-9f5a2fc44ad4.png)

This is my app: https://semar.iwgx.io/ and the repo https://github.com/iwgx/semar

Or is this not because Next? Check me if I'm wrong

If this is new Next behavior, how to disable it? Thanks

**Comment by ilhamwahabi** [comment, marked as answer]
UPDATE

After track all my previous commit, this is the commit that start this problem. Not because Next version  https://github.com/iwgx/semar/commit/56f3bb4280b738fd6aa68926a2f0239c25b6c8a0

```js
// fix `Reference error: window is not defined` although we not use SSR 
if (typeof window === 'undefined') return <div />

return (
```

But If I don't do that my app will get error, or is there another way to handle it? Thanks

**Reply by ilhamwahabi** [reply]
For now Iooks like I should using conditional in every code instead of `_app.tsx`

**Reply by raphaelbadia** [reply]
if you don't want to use SSR, maybe you can import dynamically your components with ssr:false option

```
const DynamicComponentWithNoSSR = dynamic(
  () => import('../components/hello3'),
  { ssr: false }
)
```

https://nextjs.org/docs/advanced-features/dynamic-import
